boot/freeldr/freeldr/ntldr/arch/i386/winldr.c

```diff
@@ -185,7 +185,7 @@ MempSetupPaging(IN PFN_NUMBER StartPage,
         }
 
         PhysicalPT[Page & 0x3ff].PageFrameNumber = Page;
-        PhysicalPT[Page & 0x3ff].Valid = (Page != 0);
+        PhysicalPT[Page & 0x3ff].Valid = 1;
```

Comment:
@ThFabba , @tkreuzer , could you please double-check this?

Reply:
@maharmstone Can you explain the change, please?

Reply:
@tkreuzer, clue's in the description of the commit: "Map first page of memory"